docs/installation.rst

```diff
@@ -70,7 +74,7 @@ We suggest to create a mss user.
 * login as mss user
 * create a *src* directory in /home/mss
 * cd src
-* get `mambaforge <https://github.com/conda-forge/miniforge#mambaforge>`__
+* get `miniforge <https://github.com/conda-forge/miniforge#download>`__
 * set execute bit on install script
```

Comment:
we should write miniforge, Miniforge  same way. When the word is used we should better write it exactly how they use it currently. Can you change it to Miniforge3.

The problem here is, that the software is used by people not heard about python and have not installed something before. They may match the word on the linked paged. 

 
I can tell stories what had gone wrong in the past.